### DISCUSSION TITLE
win10 宋体-简字体问题

### DISCUSSION THREAD

**Discussion by ykiZheng** [Keldos-Li/typora-latex-theme#44] [General]
使用 win10 下载仓库中附带的宋体-简字体，在 typora 和 word 中输出”回“字是黑框。
不知道是字体问题，还是非 mac 平台的原因。
如图：
<img width="191" alt="屏幕截图 2021-06-13 094939" src="https://user-images.githubusercontent.com/57142943/121792870-b60f1080-cc2c-11eb-89a8-4428e3de8a96.png">

**Comment by Keldos-Li** [comment]
很有可能是字体问题。我使用了Glyphs对字体名称和相关信息进行了设置，然后重新打包了字体文件。或许其中出现了错误。
我回头重新看看_(:з」∠)_

在问题解决之前建议用其他宋体字_(:з」∠)_

**Comment by Cpaulyz** [comment]
![image](https://user-images.githubusercontent.com/43774645/122662783-5bdff380-d1c8-11eb-9424-ef02248dead2.png)
除了回 其他有包括回的字也会有一样的问题

**Comment by Keldos-Li** [comment]
试试下载这里的系列字体，然后在userCustom.css那里替换 ，看看效果？
https://ctan.org/pkg/fandol

**Comment by Keldos-Li** [comment]
> 试试下载这里的系列字体，然后在userCustom.css那里替换 ，看看效果？
> https://ctan.org/pkg/fandol

主要是宋体字，个人觉得那个黑体很丑_(:з」∠)_

**Comment by ykiZheng** [comment]
替换后问题解决了。
但是还是建议换个宋体，这一款字体在阅读的时候横向笔画过细，略伤眼。
如下：
![图片](https://user-images.githubusercontent.com/57142943/123511284-04072800-d6b3-11eb-802b-3c618e597900.png)


> 
> 
> > 试试下载这里的系列字体，然后在userCustom.css那里替换 > ，看看效果？
> > https://ctan.org/pkg/fandol
> 
> 主要是宋体字，个人觉得那个黑体很丑_(:з」∠)_

**Comment by Keldos-Li** [comment]
> 替换后问题解决了。
> 但是还是建议换个宋体，这一款字体在阅读的时候横向笔画过细，略伤眼。
> 如下：
> ![图片](https://user-images.githubusercontent.com/57142943/123511284-04072800-d6b3-11eb-802b-3c618e597900.png)
> 
> > > 试试下载这里的系列字体，然后在userCustom.css那里替换 > > > ，看看效果？
> > > https://ctan.org/pkg/fandol
> > 
> > 
> > 主要是宋体字，个人觉得那个黑体很丑_(:з」∠)_

😢😢😢

**Comment by Keldos-Li** [comment]
刚考完，重新打包的字体放在这里 https://wws.lanzoui.com/i4gnUr1i7le

过几天再更新项目吧

**Comment by Keldos-Li** [comment]
> 刚考完，重新打包的字体放在这里 https://wws.lanzoui.com/i4gnUr1i7le
> 
> 过几天再更新项目吧

忘了说了，新的字体文件打包已经放在[另一个仓库了](https://github.com/Keldos-Li/typora-latex-theme-fonts)，然后也都给了官方的链接。

**Comment by xiaoxuan-yu** [comment]
> 替换后问题解决了。 但是还是建议换个宋体，这一款字体在阅读的时候横向笔画过细，略伤眼。 如下： ![图片](https://user-images.githubusercontent.com/57142943/123511284-04072800-d6b3-11eb-802b-3c618e597900.png)
> 
> > > 试试下载这里的系列字体，然后在userCustom.css那里替换 > > > ，看看效果？
> > > https://ctan.org/pkg/fandol
> > 
> > 
> > 主要是宋体字，个人觉得那个黑体很丑_(:з」∠)_

直接换Source Han Serif，开源没版权问题还美观（x

**Comment by RalXYZ** [comment]
> > 替换后问题解决了。 但是还是建议换个宋体，这一款字体在阅读的时候横向笔画过细，略伤眼。 如下： ![图片](https://user-images.githubusercontent.com/57142943/123511284-04072800-d6b3-11eb-802b-3c618e597900.png)
> > > > 试试下载这里的系列字体，然后在userCustom.css那里替换 > > > > > > ，看看效果？
> > > > https://ctan.org/pkg/fandol
> > > 
> > > 
> > > 主要是宋体字，个人觉得那个黑体很丑_(:з」∠)_
> 
> 直接换Source Han Serif，开源没版权问题还美观（x

感谢反馈。现在的情况是这样的：  
- @Keldos-Li 对思源宋体作为默认字体并不满意，原因请参见[这里](https://github.com/Keldos-Li/typora-latex-theme/issues/20#issuecomment-950306502)
- 目前 #36 中增加的对 Linux 的支持，默认字体是思源系的字体；具体原因参见[这里](https://github.com/Keldos-Li/typora-latex-theme/issues/20#issuecomment-955203634)

**Comment by xiaoxuan-yu** [comment]
> > > 替换后问题解决了。 但是还是建议换个宋体，这一款字体在阅读的时候横向笔画过细，略伤眼。 如下： ![图片](https://user-images.githubusercontent.com/57142943/123511284-04072800-d6b3-11eb-802b-3c618e597900.png)
> > > > > 试试下载这里的系列字体，然后在userCustom.css那里替换 > > > > > > > > > > ，看看效果？
> > > > > https://ctan.org/pkg/fandol
> > > > 
> > > > 
> > > > 主要是宋体字，个人觉得那个黑体很丑_(:з」∠)_
> > 
> > 
> > 直接换Source Han Serif，开源没版权问题还美观（x
> 
> 感谢反馈。现在的情况是这样的：
> 
> * @Keldos-Li 对思源宋体作为默认字体并不满意，原因请参见[这里](https://github.com/Keldos-Li/typora-latex-theme/issues/20#issuecomment-950306502)
> * 目前 [Add Linux support, fix bugs #36](https://github.com/Keldos-Li/typora-latex-theme/pull/36) 中增加的对 Linux 的支持，默认字体是思源系的字体；具体原因参见[这里](https://github.com/Keldos-Li/typora-latex-theme/issues/20#issuecomment-955203634)

关于思源宋体的Bold字重问题，我赞同@Keldos-Li的观点，但是在汉语环境中可以参考CTeX的方案使用黑体作为bfseries的替代，而华文宋体本身实际上也很难让人满意...

如果不考虑授权问题的话，我个人认为方正新书宋是更好的选择。

**Comment by Keldos-Li** [comment]
> > > > 替换后问题解决了。 但是还是建议换个宋体，这一款字体在阅读的时候横向笔画过细，略伤眼。 如下： ![图片](https://user-images.githubusercontent.com/57142943/123511284-04072800-d6b3-11eb-802b-3c618e597900.png)
> > > > > > 试试下载这里的系列字体，然后在userCustom.css那里替换 > > > > > > > > > > > > > > > ，看看效果？
> > > > > > https://ctan.org/pkg/fandol
> > > > > 
> > > > > 
> > > > > 主要是宋体字，个人觉得那个黑体很丑_(:з」∠)_
> > > 
> > > 
> > > 直接换Source Han Serif，开源没版权问题还美观（x
> > 
> > 
> > 感谢反馈。现在的情况是这样的：
> > 
> > * @Keldos-Li 对思源宋体作为默认字体并不满意，原因请参见[这里](https://github.com/Keldos-Li/typora-latex-theme/issues/20#issuecomment-950306502)
> > * 目前 [Add Linux support, fix bugs #36](https://github.com/Keldos-Li/typora-latex-theme/pull/36) 中增加的对 Linux 的支持，默认字体是思源系的字体；具体原因参见[这里](https://github.com/Keldos-Li/typora-latex-theme/issues/20#issuecomment-955203634)
> 
> 关于思源宋体的Bold字重问题，我赞同@Keldos-Li的观点，但是在汉语环境中可以参考CTeX的方案使用黑体作为bfseries的替代，而华文宋体本身实际上也很难让人满意...
> 
> 如果不考虑授权问题的话，我个人认为方正新书宋是更好的选择。

是的，方正新书宋应该是现在正文宋体的最好选择。我自己日常也经常使用方正新书宋（毕竟是教材和公文要求使用的字体）……

但从字体安装率来看，知道并使用方正新书宋的学生并不多…而且方正对它同样没有做一整套字重的设计；相比之下，华文宋体是中文版Office安装时附带的字体，大家电脑中基本都会有（应该？），作为最后一级fallback的衬线体足够可用了。（华文宋体装机率肯定比思源系列高。）

目前推荐的宋体字体放在[这个仓库](https://github.com/Keldos-Li/typora-latex-theme-fonts/tree/main/Windows/#宋体-简)，是把华文书宋和华文粗宋打包成字体家族的，效果会比华文宋体好些。

我**不认为**把常规语境下的“黑体”用作中文粗体系列是个好主意。CTeX的默认方案明显是早期没有高字重宋体时的妥协产物，现在较新的中文LaTeX模板也早已使用其他的方案解决粗体问题。对于排版来说，混用有衬线和无衬线字体仅来达成“粗体”效果会造成信息层级的混乱，我自己是不能接受的。

不过反正现在自定义字体的留空都有了，改成自己喜欢的字体也很方便……最终用什么宋体就留给用户自己决定吧。

**Comment by xiaoxuan-yu** [comment]
> > > > > 替换后问题解决了。 但是还是建议换个宋体，这一款字体在阅读的时候横向笔画过细，略伤眼。 如下： ![图片](https://user-images.githubusercontent.com/57142943/123511284-04072800-d6b3-11eb-802b-3c618e597900.png)
> > > > > > > 试试下载这里的系列字体，然后在userCustom.css那里替换 > > > > > > > > > > > > > > > > > > > > > ，看看效果？
> > > > > > > https://ctan.org/pkg/fandol
> > > > > > 
> > > > > > 
> > > > > > 主要是宋体字，个人觉得那个黑体很丑_(:з」∠)_
> > > > 
> > > > 
> > > > 直接换Source Han Serif，开源没版权问题还美观（x
> > > 
> > > 
> > > 感谢反馈。现在的情况是这样的：
> > > 
> > > * @Keldos-Li 对思源宋体作为默认字体并不满意，原因请参见[这里](https://github.com/Keldos-Li/typora-latex-theme/issues/20#issuecomment-950306502)
> > > * 目前 [Add Linux support, fix bugs #36](https://github.com/Keldos-Li/typora-latex-theme/pull/36) 中增加的对 Linux 的支持，默认字体是思源系的字体；具体原因参见[这里](https://github.com/Keldos-Li/typora-latex-theme/issues/20#issuecomment-955203634)
> > 
> > 
> > 关于思源宋体的Bold字重问题，我赞同@Keldos-Li的观点，但是在汉语环境中可以参考CTeX的方案使用黑体作为bfseries的替代，而华文宋体本身实际上也很难让人满意...
> > 如果不考虑授权问题的话，我个人认为方正新书宋是更好的选择。
> 
> 是的，方正新书宋应该是现在正文宋体的最好选择。我自己日常也经常使用方正新书宋（毕竟是教材和公文要求使用的字体）……
> 
> 但从字体安装率来看，知道并使用方正新书宋的学生并不多…而且方正对它同样没有做一整套字重的设计；相比之下，华文宋体是中文版Office安装时附带的字体，大家电脑中基本都会有（应该？），作为最后一级fallback的衬线体足够可用了。（华为宋体装机率肯定比思源系列高。）
> 
> 目前推荐的宋体字体放在[这个仓库](https://github.com/Keldos-Li/typora-latex-theme-fonts/tree/main/Windows/#%E5%AE%8B%E4%BD%93-%E7%AE%80)，是把华文书宋和华文粗宋打包成字体家族的，效果会比华文宋体好些。
> 
> 我**不认为**把常规语境下的“黑体”用作中文粗体系列是个好主意。CTeX的默认方案明显是早期没有高字重宋体时的妥协产物，现在较新的中文LaTeX模板也早已使用其他的方案解决粗体问题。对于排版来说，混用有衬线和无衬线字体仅来达成“粗体”效果会造成信息层级的混乱，我自己是不能接受的。
> 
> 不过反正现在自定义字体的留空都有了，改成自己喜欢的字体也很方便……最终用什么宋体就留给用户自己决定吧。

感谢您的回复，非常喜欢这套主题，希望能越做越好。
